### PR TITLE
Scrub the Scrubs

### PR DESCRIPTION
**Pull request by jvc56**

### CODE REVIEW

File: pkg/profile/profile.go [hunk 1]
```diff
@@ -92,12 +112,20 @@ func (ps *ProfileService) GetUsersGameInfo(ctx context.Context, r *pb.UsersGameI
 	var infos []*pb.UserGameInfo
 
 	for _, uuid := range r.Uuids {
-		user, err := ps.userStore.GetByUUID(ctx, uuid)
-		if err == nil {
+		user, userStoreErr := ps.userStore.GetByUUID(ctx, uuid)
+		avatarUrl := user.AvatarUrl()
```
Comment (andy-k) [inline pkg/profile/profile.go:116]: remember to check `userStoreErr`, otherwise `user` may be `nil` and this may panic

Comment (jvc56) [reply pkg/profile/profile.go]: Good catch, I believe I have fixed it.

File: pkg/tournament/service.go [hunk 2]
```diff
@@ -451,41 +494,38 @@ func authenticateDirector(ctx context.Context, ts *TournamentService, id string,
...
+		}
 
-func (ts *TournamentService) GetRecentClubSessions(ctx context.Context, req *pb.RecentClubSessionsRequest) (*pb.ClubSessionsResponse, error) {
-	return ts.tournamentStore.GetRecentClubSessions(ctx, req.Id, int(req.Count), int(req.Offset))
+		_, known = knownUsers[playerTwo]
+		if !known {
+			knownUsers[playerTwo] = mod.IsCensorable(ctx, us, playerTwo)
+		}
+		if knownUsers[playerTwo] {
+			game.Players[0].Username = mod.CensoredUsername
```
Comment (domino14) [inline pkg/tournament/service.go:525]: should this index be a 1?

Comment (jvc56) [reply pkg/tournament/service.go:525]: Yes, that's my mistake.

**Pull request merged**